### PR TITLE
[docker] update for chain ID

### PR DESCRIPTION
**Pull request by christinacelee**
## Motivation

Updating docker-related files to include chain ID:
- CLI command
- Docker mint

### CODE REVIEW

**Comment by christinacelee** [comment]
we should discuss/agree how to set the environment variables for devnet and testnet
devnet should be "DEVNET" or 3, testnet should be "TESTNET" or 2

**Comment by sausagee** [comment]
I'd say DEVNET/TESTNET over the numeric. It is easier to understand when looking at the terraform config. 

BTW, I should have given you a heads up before pushing to your branch.

**Comment by christinacelee** [comment]
oh I meant where in the prod workflow we want to set the env var - rn in this PR for example, it's being set in the Dockerfile to "TESTING", which won't work for dev/testnet. I just want to confirm that each *net release will correctly set the env var

**Comment by sherry-x** [comment]
It's set here https://github.com/libra/libra/blob/master/terraform/templates/faucet.json#L11
in terraform we can make a variable and set different value accordingly

File: docker/mint/docker-run.sh [hunk 1]
```diff
@@ -6,6 +6,11 @@ set -ex
 export RUST_BACKTRACE=full
 
 declare -a params
+if [ -z "${CHAIN_ID}" ]; then
+  echo "Docker run failed to start, please set required env var CHAIN_ID"
+  exit 1
```
Comment (sherry-x) [inline docker/mint/docker-run.sh:11]: Do you need to add this parameter in the command when running the config-builder for faucet? see like 22-24

Comment (sherry-x) [reply docker/mint/docker-run.sh:11]: If not needed, I think we don't have to set the env in this script

**Pull request closed without merging**